vectordatasource/util.py

```diff
@@ -75,3 +75,21 @@ def calculate_path_major_route(fid):
 def cycling_network(tags, fid):
     # TODO: implement me! current implementation is a stub.
     return None
+
+
+def tag_yes_to_bool(str_or_none):
+    return True if str_or_none == 'yes' else None
```

Comment:
Yea, I think it makes sense to make a generic kind of function where we convert typical truthy type values to a boolean. In this case it might not matter, but it could be useful down the road. Updated in 8c2facc.